### ISSUE TITLE
Drag and drop still not working 

### ISSUE THREAD

**Issue by christubuntu** [adirh3/Fluent-Search#88]
**Describe the bug**
Since it's introduction, Drag and drop from FS to Windows Explorer (MS word, etc.) has never worked for me. 

**To Reproduce**
1. Open FS, search any file/folder 
2.  Long press any of the result and drop it to File Explorer 
3. Nothing happens. There's a crossed circle indicating error. Then FS freezes for a while 

**Expected behavior**
The file/folder should be copied to the drop destination 

**Screenshots**

**Desktop (please complete the following information):**
 - Windows 10 Version: [e.g. 20H1]
 - Fluent Search Version [e.g. 0.9.85.30]

**Additional context**
I'm not sure but I suspect the bug is related to FS not correctly differentiating a "single click" and a "click and hold". Because, when single clicking, I also briefly see that crossed circle (even though that single click is correctly registered.

**Comment by Metroid-Prime** [comment]
I agree with @Nomad-0    Issue # seventy seven matches this one. Best to compartmentalize the issues.

**Comment by adirh3** [comment]
This issue has been potentially fixed in version

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.85.40
